app/models/post.rb

```diff
@@ -132,10 +132,8 @@ def like_for(user)
 
   #############
 
-  def self.diaspora_initialize(params)
-    new(params.to_hash.stringify_keys.slice(*column_names)).tap do |new_post|
-      new_post.author = params[:author]
-    end
+  class << self
+    alias diaspora_initialize shareable_initialize
```

Comment:
I would prefer
```ruby
  def self.diaspora_initialize(params)
    shareable_initialize(params)
  end
```
instead of the `class << self`. It's basically the same, but I think its better readable.